lib/OpenQA/Worker/Jobs.pm

```diff
@@ -694,6 +695,20 @@ sub upload_images {
     return !$tx || $tx->success;
 }
 
+sub _strip_path_prefix {
+    my ($path) = @_;
+    my ($vol, $dir, $file) = File::Spec->splitpath($path);
+    my $relpath = $file;
+    my @dirs    = File::Spec->splitdir($dir);
+    for my $d (reverse @dirs) {
```

Comment:
no offense, but this logic needs some comment on what you're trying to achieve here. The commit log only gives it half way.